app/admin/teams.rb

```diff
@@ -13,4 +13,27 @@
   #   permitted << :other if params[:action] == 'create' && current_user.admin?
   #   permitted
   # end
+
+  # Columns for index page
+  index do
+    selectable_column
+    id_column
+    column :name
+    column :code
+    column 'Shadow code', :shadow_code
+    column 'LDU', :local_divisional_unit
+    column 'Offenders', sortable: :case_information_count do |team|
+      team.case_information.size
```

Comment:
I think this should be case_information_count, otherwise we will get an N+1 query?

Reply:
@starswan somewhat counterintuitively, it seems that `.size` is the method you need to use to take advantage of the counter cache – `.count` performs an SQL count regardless of counter cache.

I was seeing N+1 queries in development, which is when I went back to [the Rails guide](https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-counter-cache) and noticed it said to use `.size` for counter cache.